universitas-negeri-semarang-fakultas-matematika-dan-ilmu-pengetahuan-alam.csl

```diff
@@ -149,8 +149,8 @@
               </else>
             </choose>
           </if>
-          <else-if type="thesis">
-            <text variable="genre" suffix="."/>
+          <else-if type="thesis"><!--Note digunakan untuk mensitasi skripsi, thesis dan disertasi-->
+            <text variable="note" suffix="."/>
```

Comment:
@rmzelle, Right, Thesis should have a "Genre" field. But, how to fill Genre variable in Mendeley Desktop? Because I just found type, I did not find genre in Mendeley Desktop.
![cuplikan layar dari 2016-09-12 11-32-57](https://cloud.githubusercontent.com/assets/11894323/18424876/0e29c276-78de-11e6-80c5-8c5d4ede2d33.png)